src/NHSD.BuyingCatalogue.Solutions.Application/Commands/Hostings/HybridHostingType/UpdateHybridHostingTypeExecutor.cs

```diff
@@ -0,0 +1,29 @@
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using NHSD.BuyingCatalogue.Solutions.Application.Commands.Execution;
+using NHSD.BuyingCatalogue.Solutions.Application.Persistence;
+
+namespace NHSD.BuyingCatalogue.Solutions.Application.Commands.Hostings.HybridHostingType
+{
+    internal sealed class UpdateHybridHostingTypeExecutor : IExecutor<UpdateHybridHostingTypeCommand>
+    {
+        private readonly HostingPartialUpdater _hostingUpdater;
+
+        public UpdateHybridHostingTypeExecutor(HostingPartialUpdater hostingUpdater) =>
+            _hostingUpdater = hostingUpdater;
+
+        public async Task UpdateAsync(UpdateHybridHostingTypeCommand request, CancellationToken cancellationToken) =>
+            await _hostingUpdater.UpdateAsync(request.Id, hosting =>
+                {
+                    hosting.HybridHostingType = new Domain.Hostings.HybridHostingType
+                    {
+                        Summary = request.Data.Summary,
+                        Link = request.Data.Link,
+                        HostingModel = request.Data.HostingModel,
+                        RequiresHSCN = request.Data.RequiresHSCN.FirstOrDefault()
```

Comment:
I am thinking here that the command property RequiresHSCN should be just a string. As the only reason it is an array is because the frontend requires it, which makes this scenario a presentation issue. If we decide to change this we want to be consistent across the project.

Reply:
👍

Reply:
simplified the command property for RequiresHSCN to a string and moved the conversion from Array to string into the API library